frontend2/src/__test__/utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect, test } from "@jest/globals";
+import { ApiApi } from "../utils/types";
+import { Api, Auth } from "../utils/api";
+
+const API = new ApiApi("http://localhost:8000");
+
+//---- AUTHORIZATION ----//
+// TEST 1: Generate/Verify API Access Token
+// This test should work, called "login" in safe API //
+test("API: Generate/Verify API Access Token", async () => {
```

Comment:
let's try to rewrite this with mocking backend calls instead of actually calling the backend
link i sent in slack: https://www.digitalocean.com/community/tutorials/how-to-test-a-react-app-with-jest-and-react-testing-library#step-3-mocking-the-fetch-method

Reply:
Yep! I commented it out for now.